docs/source/guides/getting_started/index.rst

```diff
@@ -4,20 +4,23 @@ Getting Started
 Prerequisite Knowledge
 ----------------------
 
-In order to use ``ytdl-sub`` in any of the forms listed in these docs, you will need some basic knowledge.
```

Comment:
Let's move this to a separate PR, does not need to be mixed with changing makefile

Reply:
This PR is about the docs changes, see [my comment about the rebase dance](https://github.com/jmbannon/ytdl-sub/pull/1282#issuecomment-3193445450) and LMK if it's still unclear.